README.md

```diff
@@ -97,7 +97,7 @@ render() {
 
 Example of `data` object:
 
-```
+```json
```

Comment:
Hi @deadcoder0904 ,

Would you mind changing that to `js` please ? It's not supposed to follow the strict JSON format, just the JS Object format

Reply:
Cool